packages/storage-plus/src/keys.rs

```diff
@@ -74,6 +75,33 @@ impl<'a, T: PrimaryKey<'a> + Prefixer<'a>, U: PrimaryKey<'a>> PrimaryKey<'a> for
     }
 }
 
+// use generics for combining there - so we can use &[u8], PkOwned, or IntKey
+impl<'a, T: PrimaryKey<'a> + Prefixer<'a>, U: PrimaryKey<'a> + Prefixer<'a>, V: PrimaryKey<'a>>
+    PrimaryKey<'a> for (T, U, V)
+{
+    type Prefix = T;
```

Comment:
Prefix should be `(T, U)`

Reply:
Good catch. Wouldn't we want to consider both prefixes as valid? I mean, `(T, U)` is a prefix, but wouldn't `T` be another valid one?

Reply:
Sure, but look how this is used.
Maybe we need to go through this all and check the usage and document it better.

I don't have the head space to go deep in here now. Best is to leave these PRs open and I will review and merge them (or comment on them) when I have the time to check all the details.